### ISSUE TITLE
North Polar Stereographic ocean fill

### ISSUE THREAD

**Issue by bjlittle** [SciTools/cartopy#259]
Plotting with a North Pole Stereographic projection with a non-zero central longitude using a 50m ocean Natural Earth feature, results in the ocean facecolor filling the plot.

```
from matplotlib import pyplot as plt
import numpy
import cartopy.crs as ccrs
import cartopy.feature as cfeature


ax = plt.axes(projection=ccrs.NorthPolarStereo(central_longitude=-20.0))
ax.set_extent([0, 360, 40, 90], crs=ccrs.PlateCarree())

land_green = numpy.array((240, 240, 220))/256.
land = cfeature.NaturalEarthFeature(
    category='physical',
    name='land',
    scale='50m',
    facecolor=land_green)
ax.add_feature(land)

water_blue = numpy.array((152, 183, 226))/256.
ocean = cfeature.NaturalEarthFeature(
    category='physical',
    name='ocean',
    scale='50m',
    facecolor=water_blue)
ax.add_feature(ocean)

ax.gridlines()
plt.show()
```

**Comment by pelson** [comment]
Reduces to:

```
codes = np.array([1, 2, 2, 2, 2], dtype=np.uint8)
verts = np.array([[ 132.        ,  -40.        ],
                  [ 133.        ,   -6.        ],
                  [ 126.61488834,    1.09693878],
                  [ 115.        ,   -6.        ],
                  [ 132.        ,  -40.        ]])
```

**Comment by pelson** [comment]
Fixed with #545.
